.travis.yml

```diff
@@ -6,4 +6,15 @@ jdk:
 env:
   - HADOOP_VERSION=2.7.6
   - HADOOP_VERSION=2.8.4
-script: "./gradlew test --stacktrace --info -Ddyno.hadoop.bin.version=${HADOOP_VERSION}"
+
+jobs:
+  include:
+    - stage: build
+      script: "./gradlew build"
```

Comment:
`./gradlew build` will be default also run the tests, I think we want `./gradlew build -x test`